ledger-bridge.js

```diff
@@ -3,54 +3,91 @@ import 'babel-polyfill'
 
 require('buffer')
 
-import TransportU2F from '@ledgerhq/hw-transport-u2f'
```

Comment:
Since Firefox doesn't support WebUSB, we need to keep `TransportU2F`.  We should detect WebUSB support, or minimally Firefox UA :(, and then conditionally use `TransportU2F` or `WebSocketTransport`.